src/main/rules/GCI500/GCI500.json

```diff
@@ -15,4 +15,4 @@
   ],
   "ecoScore": "0.4",
   "defaultSeverity": "Minor"
-}
\ No newline at end of file
+}
```

Comment:
why do you delete this end line ? It seemed to me that it was necessary to keep the end line (old discussion with @utarwyn)

Reply:
Yes indeed, files should end with an empty line. This is both good practice and a convention in many technologies.
Check more in this discussion: https://stackoverflow.com/a/5813359